### PR TITLE
tests: Add pytest

### PR DESCRIPTION
**Pull request by tillywoodfield**

### CODE REVIEW

File: requirements_dev.in [hunk 1]
```diff
@@ -0,0 +1,3 @@
+-r requirements.in
```
Comment (odscjames) [inline requirements_dev.in:1]: Normally we  use .txt not .in here. That way we can be sure that the versions of libs the app needs are the same as are specified in requirements.txt - if .in is used there is a risk they could get out of sync if pip-compile is run on different files at different times.

Can you change that and re-run so requirements_dev.txt updates?

**Review by odscjames — CHANGES_REQUESTED** [review]
Just 1 tiny change - thank you!

File: requirements_dev.in (hunk 1 above)
Comment (tillywoodfield) [reply requirements_dev.in:1]: Ah that makes sense, will update now.

Comment (tillywoodfield) [reply requirements_dev.in]: Updated

**Review by odscjames — APPROVED** [review]

**Comment by odscjames** [comment]
Thank you!

**Pull request merged**